src/data/javascript-quiz.ts

```diff
@@ -2084,6 +2084,36 @@ const javascriptQuiz = [
     Explanation:
       "Variable names cannot start with a number or any special character.",
     Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types"
+  },
+  {
+    Question: "If const a = 2 and then a = 5 what will be the result?",
+    Answer: "A TypeError ",
+    Distractor1: "A TypeError ",
+    Distractor2: "5",
+    Distractor3: "2",
```

Comment:
The answer field and distractor field should not be the same.
Remember that distractor fields are the wrong answer.
But also, this set of answers is better 

```suggestion
    Answer: "JavaScript throws a TypeError",
    Distractor1: "JavaScript throws a SyntaxError",
    Distractor2: "The JavaScript program will crash",
    Distractor3: "The program will return null",
```